src/test/relays/TrustedRelay.t.sol

```diff
@@ -284,7 +318,28 @@ contract TrustedRelayTest is DSTest {
         assertTrue(!_tryRemoveSigners(signers));
     }
 
+    function testAddRemoveRelayers() public {
```

Comment:
Maybe we should add tests proving that `addRelayers` and `removeRelayers` will fail without `auth`. Same for BasicRelay.